### ISSUE TITLE
Could not find module `symbol-observable/ponyfill.js` imported from `symbol-observable/index`

### ISSUE THREAD

**Issue by juwara0** [mike-north/ember-symbol-observable#8]
### Problem Description

In version `0.1.2` of this add-on I am getting the following missing module error starting yesterday after cleaning out the `node_modules` and reinstalling:

```
Error: Could not find module `symbol-observable/ponyfill.js` imported from `symbol-observable/index`
at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
```

Version `1.1.0` of `symbol-observable` was released yesterday with changes to imports: https://github.com/benlesh/symbol-observable/pull/31

The new version is getting picked up due to the float: https://github.com/mike-north/ember-symbol-observable/blob/6c6bd65a161b20a9f0f308395ad0553086ca6968/package.json#L50

If I add `symbol-observable@1.0.4` (pinned) to my `package.json` I satisfy the missing module error so I think the changes in the PR above are the culprit.

I am seeing this in my add-on that is running: 
```
"ember-cli": "2.12.3"
node v7.2.0
npm 3.10.9
```

**Comment by toranb** [comment]
@juwara0 what blocker prevents you from upgrading to v1.0.0 of this addon? I see this functional because of the ^1.0.4 (correct me if I'm wrong/ or misunderstand) :)

**Comment by toranb** [comment]
@juwara0 I cut a release to fix this ;) explicitly pull down v1.0.1 and this is good 2 go my friend
